test/metric_dimension/drop_original_test.go

```diff
@@ -0,0 +1,179 @@
+//go:build !windows
+
+package metric_dimension
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+)
+
+const (
+	testNamespace = "TestDropOriginalMetrics"
+)
+
+type DropOriginalMetricsTestRunner struct {
+	test_runner.BaseTestRunner
+}
+
+type expectation struct {
+	shouldExist bool
+	metricNames []string
+	dimensions  []dimension.Instruction
+}
+
+var _ test_runner.ITestRunner = (*DropOriginalMetricsTestRunner)(nil)
+
+func (t *DropOriginalMetricsTestRunner) Validate() status.TestGroupResult {
+	var results []status.TestResult
+
+	for name, wants := range t.testCases() {
+		result := status.TestResult{Name: name, Status: status.SUCCESSFUL}
+		for _, want := range wants {
+			if err := t.validate(want); err != nil {
+				log.Printf("failed %s/%s: %v", t.GetTestName(), name, err)
+				result.Status = status.FAILED
+				break
+			}
+		}
+		results = append(results, result)
+	}
+
+	return status.TestGroupResult{
+		Name:        t.GetTestName(),
+		TestResults: results,
+	}
+}
+
+func (t *DropOriginalMetricsTestRunner) GetTestName() string {
+	return "DropOriginalMetrics"
+}
+
+func (t *DropOriginalMetricsTestRunner) GetAgentConfigFileName() string {
+	return "drop_original_metrics.json"
+}
+
+func (t *DropOriginalMetricsTestRunner) GetMeasuredMetrics() []string {
```

Comment:
Is this called anywhere for this test? Reason I asked is this doesnt have `visitor` so wondering if it impacts in any way.

Reply:
My understanding is it's more for the interface than anything. We _use_ it in our validation, but it isn't required. It is a little confusing to not have the renamed metric here, but I don't think that the validation code actually references this list

Reply:
if not used, a nit would be to return empty or []string{""not_used}